README.md

```diff
@@ -1,5 +1,5 @@
-pkg - a binary package manager for FreeBSD
-============================================
+pkg - a binary package manager for FreeBSD and OpenBSD
+========================================================
```

Comment:
This means where `pkg` is supported and not where `pkg` compiles. Obviously, FreeBSD is the only platform that supports `pkg` so far.